Refactor: Convert to new Solo Project format

jdmedlock opened the pull request:
Convert to new Solo Project format

Resolves: N/a
See also: N/a

jdmedlock committed:
Refactor: Update to add tasks to readme.md

Update to add tasks to readme.md. ZenHub tasks will be removed from
the GitHub repo once this has been PR'ed and promoted to the
`master` branch.

Resolves: N/a
See also: N/a

jdmedlock committed:
Refactor: Remove Markdown checkbox notation from API research

Remove Markdown checkbox notation from API research. This doesn't
format correctly as a sublevel item.

Resolves: N/a
See also: N/a

RavalVarun commented on readme.md:
```diff
@@ -3,23 +3,69 @@
-`#read-me-first` channel on Discord). For detailed requirements checkout the ZenHub Board that is part of this repo.
+General instructions for all Solo Projects are located in the 
+[Chingu Library](https://voyage.docs.chingu.io/prework/howwork). For detailed 
+requirements checkout the following requirements.
```
For detailed requirements, please checkout the following:

RavalVarun commented on readme.md:
```diff
@@ -3,23 +3,69 @@
-`#read-me-first` channel on Discord). For detailed requirements checkout the ZenHub Board that is part of this repo.
+General instructions for all Solo Projects are located in the 
+[Chingu Library](https://voyage.docs.chingu.io/prework/howwork). For detailed 
+requirements checkout the following requirements.
```
For detailed requirements, please checkout the following:

jdmedlock replied:
Update applied.

RavalVarun commented on readme.md:
```diff
@@ -3,23 +3,69 @@
-If you would like to add these tasks to your repo so you can track the progress of your Solo Project and so they will be 
-more readily available for reference you will need to manually add new issues to your ZenHub board and copy/paste the text
-in our issues to yours.
+      Things to look for in the [Mapbox API](https://docs.mapbox.com/api/) documentation:
```
Points to look for in the [Mapbox API](https://docs.mapbox.com/api/) documentation:

RavalVarun replied:
or Topics to look for in the [Mapbox API](https://docs.mapbox.com/api/) documentation:

jdmedlock replied:
Update applied

RavalVarun commented on readme.md:
```diff
@@ -3,23 +3,69 @@
+- Create a map component to contain the map of your city and location markers
+
+*Style*
+- [ ] You may use any style you choose. However, it should be consistent (e.g.
+font, font size, color scheme, layout, etc.).
```
Should we provide a link to any article which gives some details about consistent styling?

jdmedlock replied:
Good idea! Adding the following:
  - [Consistent Web Design](https://1stwebdesigner.com/consistent-web-design/)
  - [Why is consistency important in Web Design?](https://laceytechsolutions.co.uk/blog/importance-of-consistency-in-web-design/)

RavalVarun commented on readme.md:
```diff
@@ -3,23 +3,69 @@
+
+*Functionality*
+- Sidebar behaviors:
+  - [ ] By default all all available markers should be displayed
```
- [ ] By default all available markers should be displayed

jdmedlock replied:
Update applied

RavalVarun commented on readme.md:
```diff
@@ -3,23 +3,69 @@
+- Sidebar behaviors:
+  - [ ] By default all all available markers should be displayed
+  - [ ] Markers update in search results and on the map while the user is 
+  typing intp the search input field
```
looks like typo for "intp"

jdmedlock replied:
Update applied

RavalVarun commented on readme.md:
```diff
@@ -3,23 +3,69 @@
+  - [ ] Markers update in search results and on the map while the user is 
+  typing intp the search input field
+  - [ ] User should be able to click a button to toggle the sidebar visibility
+- The map component displays:
```
The map component should display:

jdmedlock replied:
Update applied

RavalVarun commented on readme.md:
```diff
@@ -3,23 +3,69 @@
+  - [ ] At least five markers for five unique locations or attractions in your city
+
+*Other*
+- [ ] Your repo needs to have a robust `README.md`
```
should we add the link to the article https://medium.com/chingu/keys-to-a-well-written-readme-55c53d34fe6d

jdmedlock replied:
Update applied

RavalVarun commented on readme.md:
```diff
@@ -3,23 +3,69 @@
+
+*Other*
+- [ ] Your repo needs to have a robust `README.md`
+- [ ] Make sure that there are no errors in the developer console before submitting
```
Before submitting, make sure that there are no errors in the developer console

jdmedlock replied:
Update applied

jdmedlock committed:
Fix: Apply changes from PR#23

Apply reviewer suggestions from [PR#23](https://github.com/chingu-voyages/voyage-prework-tier2-mapbox-api/pull/23)

Resolves: #23
See also: N/a
